app/assets/stylesheets/global.css.scss

```diff
@@ -1,13 +1,13 @@
 #gradient {
-  background: red; /* For browsers that do not support gradients */
+  background:  #007eee; /* For browsers that do not support gradients */
```

Comment:
Color literals like `#007eee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Use `//` comments everywhere<br>Colon after property should be followed by one space